lib/doorkeeper/rails/routes.rb

```diff
@@ -84,12 +112,10 @@ def introspect_routes(mapping)
       end
 
       def token_info_routes(mapping)
-        routes.resource(
-          :token_info,
-          path: "token/info",
-          only: [:show], as: mapping[:as],
-          controller: mapping[:controllers]
-        )
+        routes.resource :token_info,
```

Comment:
Any reason to change this? Also the code style of changes above doesn't look the same as for this changed piece of code

Reply:
agreed - no reason for this change. Maybe it was something happening on fixing newly introduced rubocop issues.